API.md

```diff
@@ -503,36 +752,119 @@ frisby.create('First test')
 ```
 
 ### only()
+
 Exclusively run this test. When `toss()` is invoked, the test is wrapped in a
 Mocha `describe.only` block instead of a `describe` block.
 
+### timeout(ms)
+
+Sets the timeout for this request.
+
+- Types: `ms`: `string`
+- Defaults: `none`
+
+```javascript
+frisby.create('Long-running request')
+    .get('http://example.com/slowthing')
+    .timeout(5000)
+    .expectStatus(200)
+    .toss()
+```
+
+This function can also be called with no parameter to return the current configured timeout (either by default, by global setup or having used this function with a parameter previously).
```

Comment:
This is consistent with the current behavior, though it makes me wonder, do we need to do this? ^^

Reply:
I found it odd. I can't see the use case from the test developer.

Reply:
Agreed. Maybe we can remove it before the 2.0.0 release.